### PR TITLE
Replace windows.confirm with Confirm Modal

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-42](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-42)

## What?
Replace system windows.confirm with custom confirm modal

## Why?
For better ux

## Some screenshots
Снимок экрана 2021-09-10 в 15 02 26
Снимок экрана 2021-09-10 в 15 09 37

## Video
https://user-images.githubusercontent.com/89198006/132841577-dbe66c37-57d5-460a-8dfb-43f85fdf35b3.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

**Comment by maxplt** [comment]
It remains to add support for shortcuts
The appearance of modal can be changed at any time

**Comment by rasmusguterstam** [comment]
Amazing, let's get that `<FadeIn>` up and then I'll approve! 😄

**Comment by maxplt** [comment]
## Add animation!!!
https://user-images.githubusercontent.com/89198006/132846951-e74a9f8f-928f-4d54-a213-cbef0bcd613f.mov

**Comment by maxplt** [comment]
## Add animation!
https://user-images.githubusercontent.com/89198006/132847620-13a88a13-fe86-4e87-987a-2e94f785e193.mov

**Review by rasmusguterstam — APPROVED** [review]
🙏

File: src/components/member/tabs/contracts-tab/contract/master-inception.tsx [hunk 1]
```diff
@@ -54,13 +56,12 @@ export const MasterInception: React.FC<{
...
               disabled={activateContractLoading}
               onClick={() => {
-                const confirmed = window.confirm(
-                  `Are you sure you want to activate this contract with master inception of ${format(
-                    activeFrom,
-                    'yyyy-MM-dd',
-                  )}?`,
-                )
-                if (confirmed) {
+                const confirmMsg = `Are you sure you want to activate this contract with master inception of ${format(
```
Comment (vonElfvin) [inline src/components/member/tabs/contracts-tab/contract/master-inception.tsx:59]: Preferably no unnecessary abbreviations
```suggestion
                const confirmMessage = `Are you sure you want to activate this contract with master inception of ${format(
```

**Review by vonElfvin — APPROVED** [review]
Super nice!

What do you think, should the default location be in the "center center" or "center top" as it is now? Maybe "center center" is slightly neater UX?

File: src/components/member/tabs/contracts-tab/contract/master-inception.tsx (hunk 1 above)
Comment (maxplt) [reply src/components/member/tabs/contracts-tab/contract/master-inception.tsx:59]: Position "center top" is relevant only for Confirm-like modals. For other modals I think usual position is center

**Pull request merged**